doozer/doozerlib/cli/scan_sources.py

```diff
@@ -198,14 +198,17 @@ def check_config_changes(self, image_meta, build_info):
             if current_digest.strip() != prev_digest.strip():
                 self.runtime.logger.info('%s config_digest %s is differing from %s',
                                          image_meta.distgit_key, prev_digest, current_digest)
-                # fetch latest commit message on branch
+                # fetch latest commit message on branch for the image metadata file
                 with Dir(self.runtime.data_dir):
-                    rc, commit_message, _ = exectools.cmd_gather('git log -1 --format=%s', strip=True)
+                    path = f'images/{image_meta.config_filename}'
+                    rc, commit_message, _ = exectools.cmd_gather(f'git log -1 --format=%s -- {path}', strip=True)
                     if rc != 0:
-                        raise IOError(f'Unable to retrieve commit message from {self.runtime.data_dir}')
+                        raise IOError(f'Unable to retrieve commit message from {self.runtime.data_dir} for {path}')
 
                 if commit_message.lower().startswith('scan-sources:noop'):
-                    self.runtime.logger.info('Ignoring digest change since commit message indicates noop')
+                    self.runtime.logger.info('Digest change not considered for rebuild since commit message indicates noop')
+                    self.add_assessment_reason(image_meta, RebuildHint(RebuildHintCode.NOOP_CONFIG_CHANGE,
```

Comment:
Why is this needed?

Reply:
Not necessary, but added it for visibility, we'll get to see this reason in the report that's generated at the end

Reply:
Removed